### PR TITLE
TODO - Interactive List

### PR DESCRIPTION
**Pull request by DammyShittu**
## In this project, the to-do list app was given some more functionalities.

- [ ]  A new JavaScript file was created to contain methods related to status updates. It was imported as a module.
- [ ]  Double click event listener was added to the `label` and an attribute of `contenteditable` was set to `true`.
- [ ] An `onchange` eventlistener was added to each checkbox with a line through on the corresponding `label` tag to show task completion.
- [ ] Value of tasks object `completed` key was  updated upon user actions.
- [ ] Data was preserved in the localStorage.

### CODE REVIEW

**Review by de-scholar — APPROVED** [review]
Hi @DammyShittu , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js [hunk 1]
```diff
@@ -12,29 +13,84 @@ const todoList = [
...
+      });
+
+      // Hide Delete Icon On Blur
+      item.addEventListener('blur', () => {
+        const last = item.parentElement.parentElement.lastElementChild;
+        last.style.display = 'none';
+        item.parentElement.parentElement.style.backgroundColor = '#fff';
+        const ellipsis = item.parentElement.parentElement.lastElementChild.previousElementSibling;
+        ellipsis.style.display = 'block';
+      });
```
Comment (rpire) [inline src/index.js:74]: What do you think about creating a new class called `invisible` with a rule of `display: none` and give the `delete-icon` this class as well?

That way you could be able `toggle` the `invisible` class to show or hide elements on your page. 🤓

**Review by rpire — COMMENTED** [review]
Your code is really well structured and it's easy to follow. Congratulations on your work! 🤓🙌

We have just one recommendation to make your code even better. Check the comment below and let us know if you like it. ♻

File: src/index.js (hunk 1 above)
Comment (DammyShittu) [reply src/index.js:74]: Thank you for the review.
I'll work on it. 👍

**Pull request merged**